ansible/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Verify Ansible versions
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./versioncheck.py
+      changed_when: false
+
+- name: Configure Waku Nodes
+  hosts: boot
+  roles:
+        - { role: open-ports,       tags: open-ports }
+        - { role: swap-file,        tags: swap-file  }
+        - { role: nim-waku,         tags: nim-waku   }
+# TODO: cleanup if not needed, also from requiremnets
+#        - { role: conn-limit,       tags: conn-limit }
+#        - { role: certbot,          tags: certbot    }
+#        - { role: waku-peers,       tags: waku-peers }
```

Comment:
Just drop `waku-peers` and `conn-limit` for now. Use of `certbot` will be necessary only if we want websockets enabled, so if we don't need websockets we don't need `certbot`. Just drop it.